index.html

```diff
@@ -1,11 +1,582 @@
+        <div class="row">
+          <div class="col-lg-4">
+            <img src="assets/img/first-medal.svg" class="medal" />
+            <h3 id="prize-title">1st Place</h3>
+            <div class="underline"></div>
+            <p class="prize-desc">
+              • <b>$250</b> Cash Prize<br>
+              • MIT Pathfinders <b>T-Shirt</b><br>         
+              • Amazon <b>Kindle</b><br>
+              • Recognition on <b>MIT</b> Site<br>  
+              • PathCheck Foundation <b>Swag</b><br> 
+            </p>
+          </div>
+          <div id="second" class="col-lg-4">
+            <img src="assets/img/second-medal.svg" class="medal" />
+            <h3 id="prize-title">2nd Place</h3>
+            <div class="underline"></div>
+            <p class="prize-desc">
+              • <b>$150</b> Cash Prize<br>
+              • MIT Pathfinders <b>T-Shirt</b><br>
+              • Recognition on <b>MIT</b> Site<br>  
+              • PathCheck Foundation <b>Swag</b><br> 
+            </p>
+          </div>
+          <div id="second" class="col-lg-4">
+            <img src="assets/img/third-medal.svg" class="medal" />
+            <h3 id="prize-title">3rd Place</h3>
+            <div class="underline"></div>
+            <p class="prize-desc">
+              • <b>$100</b> Cash Prize<br>
+              • MIT Pathfinders <b>T-Shirt</b><br>
+              • Recognition on <b>MIT</b> Site<br>
+            </p>
+          </div>
+        </div>
+        <br>
+        <br>
+        <p style="text-align: center;">In addition, every participant will receive a digital certificate from MIT Pathfinders.</p>
+      </div>
+
+    </section> End Prizes Section -->
+
+    <!-- ======= Schedule Section ======= -->
+    <section id="timeline" class="section-with-bg">
+      <div class="container" data-aos="fade-up">
+        <div class="section-header">
+          <h2>Timeline</h2>
+          <p>Here is our timeline</p>
+        </div>
+
+        <ul class="nav nav-tabs" role="tablist" data-aos="fade-up" data-aos-delay="100">
+          <li class="nav-item">
+            <a class="nav-link active" href="#day-1" role="tab" data-toggle="tab">November 28th</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#day-2" role="tab" data-toggle="tab">December 7th</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#day-3" role="tab" data-toggle="tab">December 11th</a>
+          </li>
+        </ul>
+
+        <div class="tab-content row justify-content-center" data-aos="fade-up" data-aos-delay="200">
+
+          <!-- Schdule Day 1 -->
+          <div role="tabpanel" class="col-lg-9 tab-pane fade show active" id="day-1">
+            <br>
+            <h1 class="sub-heading"><b>Essay Abstract Due!</b><br></h1>
+            <h3 class="sub-headin">Please submit your essay abstract by 11:59 PM EST on Saturday, <b>November 28th</b>. A submission form will be available soon.</h3>
```

Comment:
you can link the form here

Reply:
gonna have to figure this out - can't embed a google form when it requires uploading